### PR TITLE
Prepares for release v5.0.1

### PR DESCRIPTION
**Pull request by dkastl**
Changes proposed in this pull request:
- Updates `main` branch in order to release v5.0.1

@gtt-project/maintainer

### CODE REVIEW

File: config/locales/ja.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ ja:
   label_name: タイトル
   label_baselayer: 背景レイヤー
   label_nearby: "近く(緯度,経度)"
-  label_layer: "レイヤー種別"
+  label_layer: "レイssssssssssssssssssssssssssssヤー種別"
```
Comment (smellman) [inline config/locales/ja.yml:17]: @sanak What is this?

Comment (smellman) [reply config/locales/ja.yml:17]: @dkastl I find your commit has this bug. I send new PR.

**Review by smellman — CHANGES_REQUESTED** [review]
I send new PR for fix label.

File: config/locales/ja.yml (hunk 1 above)
Comment (dkastl) [reply config/locales/ja.yml]: @smellman , thanks for findig this typo. I didn't see this when merging the translation from Weblate. I merged your PR and it seems to be included now in this PR as well.

**Review by smellman — APPROVED** [review]
LGTM!

**Pull request merged**